controllers/GroupController.js

```diff
@@ -0,0 +1,130 @@
+import GroupModel from '../models/Group.js'
+
+export const getAll = async (req, res) => {
+    try {
+        const groups = await GroupModel.find({ user: req.userId }).exec();
+        res.json(groups);
+    } catch (error) {
+        console.log(error);
+        res.status(500).json({
+            message: 'Не вдалося отримати групи',
+        })
```

Comment:
Now you can use function `handleError`:
```suggestion
        handleError(res, 'Не вдалося отримати групу');
```